### PR TITLE
Enhancement/sumukh/autostyle

### PR DESCRIPTION
**Pull request by ACscooter**
Huzza no merge conflicts!

### CODE REVIEW

File: client/protocols/autostyle.py [hunk 1]
```diff
@@ -39,6 +40,12 @@ def run(self, messages):
 
         grading = messages['grading']
 
+        if not self.args.question:
+            log.info("Not an autostyle question")
```
Comment (Sumukh) [inline client/protocols/autostyle.py]: That message doesn't seem correct in this condition.

File: client/protocols/autostyle.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ class AutoStyleProtocol(models.Protocol):
 
     # Timeouts are specified in seconds.
     SHORT_TIMEOUT = 10
+    RETRY_LIMIT = 5
```
Comment (Sumukh) [inline client/protocols/autostyle.py]: Why is this here? Is it used anywhere? 

AFAIK it was removed in dd63e55c8fc2e9d62e412306a07479bf4796b8f1

Comment (ACscooter) [inline client/protocols/autostyle.py]: Nope... I thought it was like a thing that somebody accessing the class would use.
